pkgs/desktops/deepin/dde-kwin/default.nix

```diff
@@ -133,6 +133,7 @@ mkDerivation rec {
   passthru.updateScript = deepin.updateScript { inherit pname version src; };
 
   meta = with stdenv.lib; {
+    broken = true;
```

Comment:
Nitpick, but can we do `broken = lib.versionAtLeast kwindowsystem.version "5.18";` ? This makes it super clear why it's broken.

Reply:
Or maybe `broken = lib.versionAtLeast (lib.versions.majorMinor kwindowsystem.version) "5.18";`

Reply:
Both expressions don't work. If you find one that works I'll happily put it in.

Reply:
`broken = versionAtLeast (versions.majorMinor (getVersion kwindowsystem)) "5.18";` seems to work